descartes_core/include/descartes_core/cart_trajectory_pt.h

```diff
@@ -160,22 +160,22 @@ struct TolerancedFrame: public Frame
   {
     Eigen::Vector3d t = a.translation();
     Eigen::Matrix3d m = a.rotation();
-    Eigen::Vector3d rpy = m.eulerAngles(2,1,0);
+    Eigen::Vector3d rxyz = m.eulerAngles(0,1,2);
```

Comment:
Can you explain why this was required?  (I just want clarity on this for the future, so that we (probably just me) don't make this mistake again)